src/processor.ts

```diff
@@ -101,6 +130,12 @@ const processor = new SubstrateBatchProcessor()
     .addEvent("Redeem.RedeemPeriodChange", eventArgsData)
     .addEvent("Security.UpdateActiveBlock", eventArgsData)
     .addEvent("Tokens.Transfer", eventArgsData)
+    .addEvent("Tokens.Deposited", circulatingSupplyArgs)
+    .addEvent("Tokens.Withdrawn", circulatingSupplyArgs)
+    .addEvent("Tokens.Locked", circulatingSupplyArgs)
+    .addEvent("Tokens.Unlocked", circulatingSupplyArgs)
+    .addEvent("Tokens.Reserved", circulatingSupplyArgs)
+    .addEvent("Tokens.Unreserved", circulatingSupplyArgs)
```

Comment:
We also probably want to handle `totalIssanceSet` which may increase or decrease the total supply (i.e. due to inflation).

Reply:
I haven't seen a single event of that type on either Interlay or Kintsugi mainnets.

But more importantly, since the total issuance can change without an event of that change being emitted (as it has in the past for failed xcm transactions), squid is forced to fetch the total issuance from chain for all other events (locked, unlocked, etc) in any case.
Meaning, we're querying the chain anyway, so no need to follow that event.